homeassistant/components/cmus/media_player.py

```diff
@@ -65,33 +65,67 @@ def setup_platform(hass, config, add_entities, discover_info=None):
     name = config[CONF_NAME]
 
     try:
-        cmus_remote = CmusDevice(host, password, port, name)
+        cmus_remote = CmusRemote(server=host, port=port, password=password)
     except exceptions.InvalidPassword:
         _LOGGER.error("The provided password was rejected by cmus")
         return False
```

Comment:
Nothing is checking this return value.
```suggestion
        return
```

Reply:
Cheers - going to include this in my next commit.